.gitignore

```diff
@@ -10,3 +10,5 @@ node_modules/
 .nyc_output
 *.lock
 /tmp
+.clasprc.json
+/tests/.clasprc.json
```

Comment:
I think adding just the `.clasprc.json` would ignore the file in all directories.